apiaudio/api_request.py

```diff
@@ -41,6 +41,9 @@ def _post_request(cls, json, url=None):
 
         cls._expanded_raise_for_status(r)
 
+        if r.status_code == 408:
+            r = requests.post(url=url, headers=headers, json=json)
```

Comment:
GET, and pass the script ID.

Also _expand_raise_for_status will probably return the timeout error before this code will be reached.